### DISCUSSION TITLE
dropdown menu class dropdown-menu-end

### DISCUSSION THREAD

**Discussion by bamhost** [bootscore/bootscore#583] [Q&A]
This has had me stumped forever, and thought I had asked about it before, but a search shows that I haven't.

How or where do I add the class dropdown-menu-end in order to right align the dropdown so that dropdown menus do not go beyond the viewport? I do not see in the header.php where I can add this...I know that I am missing something.

Thank you for your help.

**Comment by crftwrk** [comment]
You have to add the dropdown alignment classes to the menu-items in backend. See https://github.com/AlexWebLab/bootstrap-5-wordpress-navbar-walker.

Solved?
